src/client/reducers/transactionsReducer.js

```diff
@@ -1,16 +1,8 @@
 const sampleData = require('../../../sampleData');
 import * as types from '../constants/actionTypes';
 
-const initialState = {
```

Comment:
personally i'm a bigger fan of an initialState definition since there's just a few lines where we detail our default store (rather than peeking into every reducer), but i think in this case that's accounted for nicely since the reducer names are now identical to their corresponding store keys